pkg/models/networks.go

```diff
@@ -77,38 +77,35 @@ type CreateNetwork struct {
 	Name                string               `json:"name" tf:"name"`
 	Description         string               `json:"description,omitempty" tf:"description"`
 	DisplayName         string               `json:"displayName,omitempty" tf:"display_name"`
-	CloudID             int                  `json:"-" tf:"cloud_id"`
-	GroupID             string               `json:"-" tf:"group_id"`
-	TypeID              int                  `json:"-" tf:"type_id"`
-	PoolID              int                  `json:"pool,omitempty" tf:"pool_id"`
-	NetworkDomainID     int                  `json:"-" tf:"domain_id"`
-	Site                IDStringModel        `json:"site,omitempty"`
-	Type                IDModel              `json:"type,omitempty"`
-	NetworkDomain       *IDModel             `json:"networkDomain,omitempty"`
-	NetworkProxy        *IDModel             `json:"networkProxy,omitempty"`
```

Comment:
why did you remove these ? We still have support for Static Pool Networks. Did the story say that we need to remove static pool networks anywhere ?

Reply:
updated